Wikipedia/Networking/Fetchers/WMFEnglishFeaturedTitleFetcher.m

```diff
@@ -0,0 +1,174 @@
+@end
+
+@interface WMFEnglishFeaturedTitleResponseSerializer : WMFApiJsonResponseSerializer
+@end
+
+@interface WMFTitlePreviewRequestSerializer : AFHTTPRequestSerializer
+@end
+
+@interface WMFEnglishFeaturedTitleFetcher ()
+@property (nonatomic, strong) AFHTTPRequestOperationManager* featuredTitleOperationManager;
+@property (nonatomic, strong) AFHTTPRequestOperationManager* titlePreviewOperationManager;
+@end
+
+@implementation WMFEnglishFeaturedTitleFetcher
+
++ (AFHTTPRequestOperationManager*)featuredTitleOperationManager {
+    AFHTTPRequestOperationManager* featuredTitleOperationManager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
+    featuredTitleOperationManager.requestSerializer  = [WMFEnglishFeaturedTitleRequestSerializer serializer];
+    featuredTitleOperationManager.responseSerializer = [WMFEnglishFeaturedTitleResponseSerializer serializer];
+    return featuredTitleOperationManager;
+}
+
++ (AFHTTPRequestOperationManager*)titlePreviewOperationManager {
+    AFHTTPRequestOperationManager* titlePreviewOperationManager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
+    titlePreviewOperationManager.requestSerializer  = [WMFTitlePreviewRequestSerializer serializer];
+    titlePreviewOperationManager.responseSerializer =
+        [WMFMantleJSONResponseSerializer serializerForValuesInDictionaryOfType:[MWKSearchResult class] fromKeypath:@"query.pages"];
+    return titlePreviewOperationManager;
+}
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        self.featuredTitleOperationManager = [WMFEnglishFeaturedTitleFetcher featuredTitleOperationManager];
+        self.titlePreviewOperationManager  = [WMFEnglishFeaturedTitleFetcher titlePreviewOperationManager];
+    }
+    return self;
+}
+
+- (AnyPromise*)fetchFeaturedArticlePreviewForDate:(NSDate*)date {
+    @weakify(self);
+    MWKSite* site = [MWKSite siteWithLanguage:@"en"];
+    return [self.featuredTitleOperationManager wmf_GETWithSite:site parameters:date]
+           .thenInBackground(^(NSString* title) {
+        @strongify(self);
+        if (!self) {
+            return [AnyPromise promiseWithValue:[NSError cancelledError]];
+        }
+        return [self.titlePreviewOperationManager wmf_GETWithSite:site parameters:title]
+        .then(^(NSArray<MWKSearchResult*>* featuredTitlePreviews) {
+            return featuredTitlePreviews.firstObject;
+        });
+    });
+}
+
+@end
+
+@implementation WMFEnglishFeaturedTitleRequestSerializer
+
++ (NSDateFormatter*)featuredArticleDateFormatter {
+    static NSDateFormatter* feedItemDateFormatter;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        feedItemDateFormatter = [[NSDateFormatter alloc] init];
+        feedItemDateFormatter.dateFormat = @"MMMM d, YYYY";
+        // feed format uses US dates—specifically month names
+        feedItemDateFormatter.locale = [NSLocale localeWithLocaleIdentifier:@"en-US"];
```

Comment:
I would ask/confirm the product and/or designers on this.

For instance, if the search site is english, but the device language is in french, then the "Continue Reading" "Random", Nearby", and "Because You read" section headers will all be in french, but this date will be the only header not localized to the device language and instead be in english.